src/applications/model/gaming-streaming-server.cc

```diff
@@ -127,7 +127,7 @@ GamingStreamingServer::Send (int idx)
 {
   NS_LOG_FUNCTION (this);
 
-  uint32_t pktSize = std::max ((int)m_iatList[idx]->GetInteger (), 12); //TODO: think about 12
+  uint32_t pktSize = std::max ((int)m_packetSizeVariables[idx]->GetInteger (), 12); //TODO: think about 12
```

Comment:
I'm still not sure about this. If we are interested in just the delay we should use the timestamp tag instead of seqTs header